concent_api/verifier/utils.py

```
@@ -0,0 +1,94 @@
+from base64 import b64encode
+import os
+import subprocess
+import zipfile
+
+import requests
+
+from django.conf import settings
+
+from golem_messages import message
+from golem_messages.shortcuts import dump
+
+from .constants import UNPACK_CHUNK_SIZE
+
+
+def clean_directory(directory_path: str):
+    """ Removes all files from given directory path. """
+    for file in os.listdir(directory_path):
+        file_path = os.path.join(directory_path, file)
+        try:
+            if os.path.isfile(file_path):
+                os.unlink(file_path)
+        except OSError:
+            pass
```

Any silenced errors should be logged.

---

Outdated, already done.